lib/highline.rb

```diff
@@ -989,12 +989,12 @@ def get_response(  )
   # instead.  This is to support any special handling for the final sequence.
   #
   def page_print( output )
-    lines = output.scan(/[^\n]*\n?/)
+    lines = output.lines.to_a
```

Comment:
Actually, `lines.to_a` is safer.  Line endings different on different operating systems.  `\n` is typical for Unix, but Windows systems will usually have `\r\n'.  We're better off to let Ruby cover those differences for us.